package practice;

public class ExceptionHandling {

	public static void main(String[] args) {
		 int a=10,b=0;
		 
		 int  c=0;
	try	 {
		int arr[]=null;
		System.out.println(arr[1]);
			c=a/b; 
		 }
	catch(Exception e)
	{
		System.out.println("error occure");
	}
	System.out.println(c);
	System.out.println("end of the program");
		
		
	
	}

}
